common/pulp/common/constants.py

```diff
@@ -59,13 +59,13 @@
 # Constant used to determine whether a CeleryBeatLock should be removed due to age
 CELERYBEAT_LOCK_MAX_AGE = 200
 # The amount of time in seconds before a Celery process is considered missing
-CELERY_TIMEOUT_SECONDS = 300
+CELERY_TIMEOUT_SECONDS = 60
```

Comment:
This will change actual failover to be < 60 seconds for Pulp usage not involving `pulp-manage-db`. I like this change, but we should probably do it as a separate commit. It also needs to come with its own changes including docs updates and a dedicated release note. Is making this change with 2.12 something we want to do?